client/src/App.css

```diff
@@ -1,3 +1,15 @@
+*,
```

Comment:
Have you considered using styled-components? Then you can skip having .css files and can incorporate the css directly into the react components. Then you could set up a globalstyles file where you can import the font-family and set the basic settings such as box-sizing, font-size, colors and so on for all elements (will save you repeat css code).

Reply:
Yes, I will do that in the next steps. We made up the template with storybook and then there come the css files as default. 
Thanks for review! 😊